src/events.d.ts

```diff
@@ -169,3 +169,8 @@ type ExternalWorkflowExecutionSignaledEvent = BaseEvent &
   Pick<HistoryEvent, 'externalWorkflowExecutionSignaledEventAttributes'>;
 type UpsertWorkflowSearchAttributesEvent = BaseEvent &
   Pick<HistoryEvent, 'upsertWorkflowSearchAttributesEventAttributes'>;
+declare namespace svelte.JSX {
```

Comment:
I don't love this, but if we're going to do it, then it should go in `global.d.ts` and _not_ `events.d.ts`.